client/.eslintrc.json

```diff
@@ -18,6 +18,27 @@
         }
     },
     "rules": {
-        "no-underscore-dangle": [2, { "allowAfterThis": true }]
+        "no-underscore-dangle": "off",
+        "no-param-reassign": ["error", { "props": false }],
+        "react/function-component-definition": [
+            2,
+            {
+                "namedComponents": "arrow-function",
+                "unnamedComponents": "arrow-function"
+            }
+        ],
+        "no-console": "off",
+        "no-alert": "off",
+        "@typescript-eslint/no-unused-vars" : "off",
+        "no-nested-ternary": "off",
+        "import/prefer-default-export": "off",
+        "react/require-default-props": "off",
+        "react/no-array-index-key": "off",
+        "react/jsx-props-no-spreading": "off",
+        "react/no-unstable-nested-components": [
+            "error",
+            { "allowAsProps": true }
+        ],
+        "jsx-a11y/anchor-is-valid": "off"
```

Comment:
Some of them are helpful like with react/function-component-definition, it would force react functional components to use the function keyboard instead of arrow functions. i had to disable no-unused-vars because the linter thought a variable was being unused but it was being used as a prop. I feel like some of these could be removed but i didnt know the best way to change some of the code

Reply:
I think some of the lints are worth disabling. Things like no-array-index-key don't have many easy alternatives (to my knowledge). This list looks reasonable to me, we can try to make no more exceptions going forward :+1: